### ISSUE TITLE
Fonts

### ISSUE THREAD

**Issue by gus-quintini** [gus-quintini/gus-quintini.ca#2]
@thomasjbradley

Hey Thomas I just checked my website today and they're not loading for some reason. I tried to figure it out myself but I couldn't figure it out. When I check on the inspector it says invalid property value.

**Comment by thomasjbradley** [comment]
In the `main.css`, try getting rid of the `..` part for the font URLs, they should read just: `/fonts…`

**Comment by gus-quintini** [comment]
Thanks! I'll try that, ill let you know how it goes :)

**Comment by gus-quintini** [comment]
It didn't seem to fix it. The weird thing is that my websites fonts are absolutely fine on Jekyll but they're not there when its live. I also haven't touched my website in a while, so I'm not sure why this is happening out of nowhere. 

@thomasjbradley

**Comment by gus-quintini** [comment]
just fixed it! sorry for all the comments haha. For some reason it wasn't liking me putting normal beside the name of the font

**Comment by thomasjbradley** [comment]
Makes sense because `normal` isn’t a `font-family` it’s a `font-weight`.
